Lanthanum.Web/Views/Footer/ContactUs.cshtml

```diff
@@ -0,0 +1,63 @@
+﻿@{
+    ViewData["Title"] = "Contact Us";
+}
+
+<div class="container py-2">
+    <h2 class="contact-us-title text-center">How can we help you?</h2>
+</div>
+
+<div class="container contact-form py-2">
+    <form class="mx-5">
+        <div class="row mx-5">
+            <div class="col-md-6">
+                <div class="form-group">
+                    <label for="input-name" class="contact-us-user-info-field-title">Your name</label>
+                    <input type="text" class="form-control contact-us-input" id="input-name" placeholder="How should we call you?">
+                </div>
+                <div class="form-group">
+                    <label for="input-email" class="contact-us-user-info-field-title">Email</label>
+                    <input type="email" class="form-control contact-us-input" id="input-email" placeholder="We’ll get back to you within several hours or sooner.">
+                </div>
+                <div class="form-group">
+                    <label for="input-phone" class="contact-us-user-info-field-title">Phone number</label>
+                    <input type="text" class="form-control contact-us-input" id="input-phone" placeholder="How should we call you?">
+                </div>
+
+            </div>
+            <div class="col-md-6">
+                <div class="form-group">
+                    <label for="inputLastname" class="contact-us-user-info-field-title">What we need to know</label>
+                    <textarea name="txtMsg" class="form-control contact-us-input-message" placeholder="Write us a message" style="width: 100%; height: 184px;"></textarea>
```

Comment:
Move styles to css file

Reply:
If move the styles to CSS-file, they won't work. So current code is best choice.